Lib/imaplib.py

```diff
@@ -110,7 +110,7 @@
         br'"')
 # Literal is no longer used; kept for backward compatibility.
 Literal = re.compile(br'.*{(?P<size>\d+)}$', re.ASCII)
-MapCRLF = re.compile(br'\r\n|\r|\n')
+MapCRLF = re.compile(br'[\r\n]')
 # We no longer exclude the ']' character from the data portion of the response
```

Comment:
I don't think this is correct. `MapCRLF` is only used in one line, as far as I can tell, to normalize all line endings:
```
literal = MapCRLF.sub(CRLF, message)
```

With the new pattern, `b'ab\r\ncd\r\n'` is replaced with `b'ab\r\n\r\ncd\r\n\r\n'`. The old pattern leaves that string unchanged and correctly replaces `b'ab\ncd\n'` with `b'ab\r\ncd\r\n'`.

Reply:
Thanks for pointing this out - I guess I should just take out the substitution, and add a test for it, so as to fix bpo5430 as well.